### PR TITLE
Bionic Claws cutting vs. stabbing change

### PR DESCRIPTION
**Pull request by i2amroy**
The bionic description speaks of "considerable cutting damage", when previously they did mostly bashing and stabbing damage. This stat change makes them into much more wolverine-esque claws, used for slicing, not necessarily for stabbing.

### CODE REVIEW

**Comment by kevingranade** [comment]
I'd prefer switching the description, claws don't mean cutting, they mean mostly piercing.  We JUST removed the cut tool quality from these because it doesn't make sense for claws.

**Comment by i2amroy** [comment]
I can see not having the cut tool quality just due to unwieldiness, but I'd like to point out that the vast majority of times you see "wolverine claws" in films, etc. they aren't being used to stab, they are being used to cut. Honestly I think it might be better to keep them as cutting things and then have a bionic dagger or something similar that is designed for stabbing, though if not having any cutting capability with them is really what we want then it's fine too.

**Comment by kevingranade** [comment]
They aren't "wolverine claws", they'd be more like bagh nakh in scale.
There simply isn't a good way to attach blades to a human hand that would be good for cutting, stabbing or clawing on the other hand would work fine.

**Comment by i2amroy** [comment]
Actually a feasible design for slashing would be pretty simple. Blades are normally present in a sheath that runs the length of the forearm. Right before you extend the blades, and lasting for as long as they are out, your wrist is locked into a straight position (fingers curled into a fist). Blades slide up out of your arm, extending a good 5 or so inches beyond the closed fingers, and then lock into place with the sheath, which has remained stationary and is basically strapped around the bones in your forearm. Then just pretend that you are slapping the person except that your fist is closed instead of open. The blades should cut right into whatever you'd hit and any force is transmitted directly into the bones of your forearm (and in such a way you could actually hit harder than with a slap, since it skips the weaker wrist joint completely, plus a *much* smaller contact area).

Basically [this](https://www.youtube.com/watch?v=sdcGek-NoFQ#t=98), but sharper and mounted internally on the back/around the arm instead of being an external thing attached with straps. He even slashes his way through a sheet of cardboard shortly after the 1 minute mark, and then shortly afterward goes right through a watermelon. It can be done.

**Comment by chaosvolt** [comment]
Did we at least retain the butcher quality?

Regardless, I'd always thought bionic claws were a tad odd. They're designed for human use, yet are far less practical in design compared to randomly-acquired claw-based mutations that have little reason to comform to the needs of a human.

Though it is understandable, as in gameplay terms losing the ability to pick stuff up via mutation would be infinitely more crippling than a restriction you can disable at will.

However, from what I'd seen bionic claws exclusively affect the piercing skill. Would they correctly recognize their having the "unarmed weapon" flag and rely on unarmed skill if they did cutting damage instead?

**Comment by Soadreqm** [comment]
What's the practical difference between slashing and stabbing damage, anyway? Armor piercing? I think it's less to do with the size of the weapon, and more with how you use it. Like, tiger claws do slashing damage, because you can only slash with them. The monomolecular blade bionic (probably something like the Adam Jensen arm blade?) is much bigger, but a stabbing weapon, because you can thrust with it.

I think Wolverine's claws would probably be a stabbing weapon in Cata terms. He mostly uses them for slashing, but they're also usable for punching and impaling things. If the bionic claws aren't like Wolverine's claws, the question would whether they're more like small, curved blades that you rake along a zombie's skin, or long, straight blades that you punch through a zombie's chest.

As for the UNARMED_WEAPON flag, I think that's just supposed to make weapons usable with unarmed fighting styles, not affect the skill used.

**Comment by chaosvolt** [comment]
You would expect them to be more like small cutting claws, but that assumes they're...well, small. Enough that you'd be able to pick stuff up. Since you can't, I suppose it's reasonably that they're more like impaling claws.

It does mean that we have a dearth of variety in bionic weapons though.We have...maybe cutting for fingertip razors (assuming theeir bonus damage actually factors in any weapon skills), then 2 piercing weapons, and one rather derpy ranged weapon that, again, I have no idea about what skill it uses.

**Comment by chaosvolt** [comment]
Sad dragon ensues. ;w;
